INSTALL.md

```diff
@@ -93,6 +93,44 @@ $ make -f policy_utilities.mak clean
 $ make -f policy_utilities.mak
 ```
 
+If you wish to use the Policy Generator instead of manually composing the
+policies, you need to build it in utilities. The Policy Generator requires the
+json-schema-validator library which in turn depends on the JSON for Modern C++
+library. Follow the following instructions to have them built and installed on
```

Comment:
Maybe leave a blank line before the last sentence, and reword it as:

Follow these instructions ...

Otherwise, `Follow the following` sounds a bit odd to read.

Reply:
Done